### ISSUE TITLE
LibVLC causing crashes

### ISSUE THREAD

**Issue by ultrawidegamer** [Yellow-Dog-Man/Resonite-Issues#2962]
### Describe the bug?

Whenever im in he Navy Seal world there are constant crashes when listening to music streams and have determined that it is caused by LibVLC. 

### To Reproduce

load up any long form of video stream and listen to it till it causes issues (10-180mins usually)

### Expected behavior

playback to work without crashing

### Resonite Version Number

2024.9.13.672

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Quest 2

### Log Files

[error.log](https://github.com/user-attachments/files/17007110/error.log)

### Additional Context

This issue might be resolved by this issue
https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1476

### Reporters

ultrawidegamer
navy3001

**Comment by GrayBoltWolf** [comment]
We've noticed this in #2838, #2845, #2890 as well, video players arbitrarily crash Resonite both Unity and VLC.

**Comment by shiftyscales** [comment]
I agree it's likely this issue is related to / a duplicate of one of the several linked issues.

As far as I recall, we don't really have much option for modifying the code of the Unity Native or VLC players. This solution for these issues likely will be #1476 so we aren't using an old, crusty library, and have more flexibility on the integration.
